### PR TITLE
show tooltips even when the button is disabled. #3326

### LINKED ISSUE #3326: show tooltips even when the button is disabled.
It may be useful to show the tooltip even when the button is disabled. Currently, the tooltip shows up only if the button is enabled.

### PR DESCRIPTION
**Pull request by MrFool**
Fixes #3326

### CODE REVIEW

**Comment by MrFool** [comment]
screen shot 2015-04-20 at 12 59 26 am

Sample, hovering on a disabled button on instructor home page.

**Comment by MrFool** [comment]
Might have to use the official bootstrap method of wrapping them buttons in div for it to work properly since this workaround causes the button to get indented into the page upon hovering as it explicitly removes suppression of on hover events.

In that case, this might be a larger issue as it requires significant change of html being appended for all buttons but is definitely possible.

**Comment by MrFool** [comment]
samplepublish
samplesubmit
sampleremind

Sample for the three buttons that may be disabled on home page, the AJAX loaded buttons on sessions page will work eventually but needs to be fixed via #3394 

In general,

screen shot 2015-05-13 at 11 34 37 am

File: src/main/java/teammates/ui/controller/PageData.java [hunk 1]
```diff
@@ -736,41 +736,98 @@ public String getInstructorFeedbackSessionActions(FeedbackSessionAttributes sess
...
+        
+        if (hasSubmit) {
+            result.append(
+                    "<a class=\"btn btn-default btn-xs btn-tm-actions session-submit-for-test" + "\" " +
+                    "href=\"" + getInstructorFeedbackSessionSubmitLink(session.courseId, session.feedbackSessionName) + "\" " +
+                    "title=\"" + Const.Tooltips.FEEDBACK_SESSION_SUBMIT + "\" data-toggle=\"tooltip\" data-placement=\"top\"" +
+                    disableSubmitSessionStr +">Submit" +
+                    "</a> "
+                );
+        } else {
```
Comment (kanghj) [inline src/main/java/teammates/ui/controller/PageData.java]: Will it be better to just wrap the button in a div, and add the tooltip to the div, for both cases?

Comment (MrFool) [inline src/main/java/teammates/ui/controller/PageData.java]: Most likely, we can drop a branch that way

Comment (MrFool) [inline src/main/java/teammates/ui/controller/PageData.java]: I change back to ongoing and work on that and see how many of these branches I can drop

**Comment by MrFool** [comment]
Thanks for the idea, managed to remove all the branching

**Pull request merged**